tests/conftest.py

```diff
@@ -361,6 +361,18 @@ def __init__(
         self.redis_sock = os.path.join(self.mount, "redis.sock")
         self.redis_chassis_sock = os.path.join(self.mount, "redis_chassis.sock")
 
+        # Initialize the database configuration
+        with open("/var/run/redis/sonic-db/database_config.json") as f:
+            db_conf = json.load(f)
+
+        db_conf["INSTANCES"]["redis"]["unix_socket_path"] = self.redis_sock
+        vs_db_conf_file = f"/tmp/database_config-{self.pid}.json"
+
+        with open(vs_db_conf_file, "w") as f:
+            json.dump(db_conf, f)
```

Comment:
Why not create a vs_db_conf_file under `tests` directory, as source code, and initialize from it directly?